### PR TITLE
Announce opam 2.2.0~beta3

### PR DESCRIPTION
**Pull request by kit-ty-kate**
To merge after https://github.com/ocaml/opam/pull/6004

### CODE REVIEW

File: opam-2-2-0-beta3.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+This version is a beta, we invite users to test it to spot previously
+unnoticed bugs as we near the stable release.
+
+## Changes
+
+* Greatly improve the opam init user experience on Windows
+* New option `opam init --cygwin-extra-packages=CYGWIN_PKGS --cygwin-internal-install`,
+  to specify additional packages for the internal Cygwin
+* Redirect the opam root to C:\opamroot when the opam root contains spaces on Windows
```
Comment (rjbou) [inline opam-2-2-0-beta3.md:26]:
```suggestion
* Redirect the opam root to `C:\opam-xxx` when the opam root contains spaces on Windows
```

File: opam-2-2-0-beta3.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+* Out-of-the-box UTF-8 paged --help on Windows
```
Comment (rjbou) [inline opam-2-2-0-beta3.md:27]:
```suggestion
* Out-of-the-box UTF-8 paged `--help` on Windows
```

File: opam-2-2-0-beta3.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+* Fix a performance regression when calling opam install --deps-only on an already installed package
```
Comment (rjbou) [inline opam-2-2-0-beta3.md:28]:
```suggestion
* Fix a performance regression when calling `opam install --deps-only` on an already installed package
```

File: opam-2-2-0-beta3.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+* Fix several edge cases related to environment reverting
+* Fixed some issues that could appear when upgrading from previous versions of opam
+* Fix various issues with opam tree --with-test
```
Comment (rjbou) [inline opam-2-2-0-beta3.md:31]:
```suggestion
* Fix various issues with `opam tree --with-test`
```

File: opam-2-2-0-beta3.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+This version is a beta, we invite users to test it to spot previously
+unnoticed bugs as we near the stable release.
+
+## Changes
+
+* Greatly improve the opam init user experience on Windows
+* New option `opam init --cygwin-extra-packages=CYGWIN_PKGS --cygwin-internal-install`,
+  to specify additional packages for the internal Cygwin
+* Redirect the opam root to `C:\opamroot-xxx` when the opam root contains spaces on Windows
```
Comment (kit-ty-kate) [inline opam-2-2-0-beta3.md:26]:
```suggestion
* Redirect the opam root to `C:\opamroot\xxx` when the opam root contains spaces on Windows
```

File: opam-2-2-0-beta3.md [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+[release note](https://github.com/ocaml/opam/releases/tag/2.2.0-beta3).
+This release also includes PRs improving the documentation and improving
+and extending the tests.
+
+## Try it on Windows!
+
+Now that the [Windows support was merged in opam-repository](https://github.com/ocaml/opam-repository/pull/25861),
+testing is as simple as calling the following command from a PowerShell terminal:
+```
+Invoke-Expression "& { $(Invoke-RestMethod https://raw.githubusercontent.com/ocaml/opam/master/shell/install.ps1) }"
```
Comment (kit-ty-kate) [inline opam-2-2-0-beta3.md:47]:
```suggestion
Invoke-Expression "& { $(Invoke-RestMethod https://raw.githubusercontent.com/kit-ty-kate/opam/windows-installer/shell/install.ps1) }"
```

File: opam-2-2-0-beta3.md [hunk 7]
```diff
@@ -0,0 +1,77 @@
...
+* Fix some failure cases when extracting tarballs which contain symlinks on Windows
+ 
+Various other general and performance improvements were made and bugs were fixed.
+API changes are denoted in the
+[release note](https://github.com/ocaml/opam/releases/tag/2.2.0-beta3).
+This release also includes PRs improving the documentation and improving
+and extending the tests.
+
+## Try it on Windows!
+
```
Comment (kit-ty-kate) [inline opam-2-2-0-beta3.md:43]:
```suggestion

**BEWARE: the command shown below is experimental, use caution and please do report any issues that you are experiencing. If you prefer to not use our experimental script, feel free to get the Windows binary directly from [the Release Page](https://github.com/ocaml/opam/releases/tag/2.2.0-beta3) and put it in your directory of choice instead.**

```

File: opam-2-2-0-beta3.md [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+**BEWARE: the command shown below is experimental, use caution and please do report any issues that you are experiencing. If you prefer to not use our experimental script, feel free to get the Windows binary directly from [the Release Page](https://github.com/ocaml/opam/releases/tag/2.2.0-beta3) and put it in your directory of choice instead.**
+
+Now that the [Windows support was merged in opam-repository](https://github.com/ocaml/opam-repository/pull/25861),
+testing is as simple as calling the following command from a PowerShell terminal:
+```
+Invoke-Expression "& { $(Invoke-RestMethod https://raw.githubusercontent.com/kit-ty-kate/opam/windows-installer/shell/install.ps1) }"
+```
+and a simple `opam init` will work out-of-the-box.
```
Comment (kit-ty-kate) [inline opam-2-2-0-beta3.md:51]:
```suggestion
opening a new terminal, and a simple `opam init` will work out-of-the-box.
```

File: opam-2-2-0-beta3.md [hunk 9]
```diff
@@ -0,0 +1,79 @@
...
+  "Raja Boujbel - OCamlPro" {"mailto:raja.boujbel(à)ocamlpro.com"}
+  "Kate Deplaix - Ahrefs" {"mailto:kit-ty-kate(à)outlook.com"}
+  "David Allsopp - Tarides" {"mailto:david(à)tarides.com"}
+]
+date: "2024-06-10"
+--BODY--
+
+_Feedback on this post is welcomed on [Discuss](https://discuss.ocaml.org/t/ann-opam-2-2-0-beta3/XXXX)!_
+
+We are indescribably excited to announce the third and final beta release of opam 2.2.0.
```
Comment (dra27) [inline opam-2-2-0-beta3.md:12]:
```suggestion
We are very pleased to announce the third and final beta release of opam 2.2.0.
```

File: opam-2-2-0-beta3.md [hunk 10]
```diff
@@ -0,0 +1,79 @@
...
+
+We've done our best to polish everything and squash as much bugs as possible in order
```
Comment (dra27) [inline opam-2-2-0-beta3.md:14]:
```suggestion
We've done our best to polish everything and squash as many bugs as possible in order
```

File: opam-2-2-0-beta3.md [hunk 11]
```diff
@@ -0,0 +1,79 @@
...
+to be ready for the final release. You can view the full list of changes in the
+[release note](https://github.com/ocaml/opam/releases/tag/2.2.0-beta3).
+
+This version is a beta, we invite users to test it to spot previously
+unnoticed bugs as we near the stable release.
+
+## Changes
+
+* Greatly improve the opam init user experience on Windows
```
Comment (dra27) [inline opam-2-2-0-beta3.md:23]:
```suggestion
* Greatly enhance the opam init user experience on Windows, and the number
  of recognised configurations
```

File: opam-2-2-0-beta3.md [hunk 12]
```diff
@@ -0,0 +1,79 @@
...
+* New option `opam init --cygwin-extra-packages=CYGWIN_PKGS --cygwin-internal-install`,
+  to specify additional packages for the internal Cygwin
```
Comment (dra27) [inline opam-2-2-0-beta3.md:25]:
```suggestion
  to specify additional packages for the internal Cygwin installation
```

**Review by dra27 — COMMENTED** [review]
Very minor nits - LGTM, thanks!
